app/_hub/kong-inc/request-transformer/how-to/_templates.md

```diff
@@ -9,10 +9,11 @@ You can use any of the current request headers, query parameters, and captured U
 groups as templates to populate supported configuration fields.
 
 | Request Parameter | Template
-| ------------- | -----------
-| header        | `$(headers.<header_name>)`, `$(headers["<Header-Name>"])` or `$(headers["<header-name>"])`)
-| querystring   | `$(query_params.<query-param-name>)` or `$(query_params["<query-param-name>"])`)
-| captured URIs | `$(uri_captures.<group-name>)` or `$(uri_captures["<group-name>"])`)
+| -------------- | -----------
+| header         | `$(headers.<header_name>)`, `$(headers["<Header-Name>"])` or `$(headers["<header-name>"])`
+| querystring    | `$(query_params.<query-param-name>)` or `$(query_params["<query-param-name>"])`
+| captured URIs  | `$(uri_captures.<group-name>)` or `$(uri_captures["<group-name>"])`
+| shared context | `$(shared.<context-key>)` or  `$(shared["<context-key>"])`
```

Comment:
```suggestion
| shared variables | `$(shared.<variable-name>)` or `$(shared["<variable-name>"])`)
```
We have this documented in the [Request Transformer Advanced](https://docs.konghq.com/hub/kong-inc/request-transformer-advanced/how-to/templates/) plugin in this format, so using it for consistency.